### PR TITLE
redirect logged in user to home via user_auth.ex and router instead o…

### PR DESCRIPTION
**Pull request by CorneliaKelinske**
…f the page controller

### CODE REVIEW

**Comment by CorneliaKelinske** [comment]
Quick check please, I think this is more elegant than before

File: lib/the_little_thinkers_space_web/router.ex [hunk 1]
```diff
@@ -20,7 +20,7 @@ defmodule TheLittleThinkersSpaceWeb.Router do
   scope "/", TheLittleThinkersSpaceWeb do
     pipe_through :browser
 
-    get "/", PageController, :index
+    # get "/", PageController, :index
```
Comment (alanvardy) [inline lib/the_little_thinkers_space_web/router.ex]: :scissors:

File: lib/the_little_thinkers_space_web/router.ex [hunk 2]
```diff
@@ -62,6 +62,7 @@ defmodule TheLittleThinkersSpaceWeb.Router do
   scope "/", TheLittleThinkersSpaceWeb do
     pipe_through [:browser, :redirect_if_user_is_authenticated]
```
Comment (alanvardy) [inline lib/the_little_thinkers_space_web/router.ex:63]: :+1:

File: lib/the_little_thinkers_space_web/controllers/page_controller.ex [hunk 3]
```diff
@@ -2,13 +2,11 @@ defmodule TheLittleThinkersSpaceWeb.PageController do
...
   def index(conn, _params) do
-    if conn.assigns[:current_user] do
-      render(conn, "home.html")
-    else
-      render(conn, "index.html")
-    end
+    render(conn, "index.html")
   end
 
+  # end
```
Comment (alanvardy) [inline lib/the_little_thinkers_space_web/controllers/page_controller.ex]: :scissors:

**Review by alanvardy — APPROVED** [review]
Looks good!

**Pull request merged**